admin/templates/base.html

```diff
@@ -0,0 +1,109 @@
+{% load staticfiles %}
+<script src="{% static 'vendor/bower_components/raven-js/dist/raven.min.js' %}"></script>
+<script src="{% static 'vendor/bower_components/jquery/dist/jquery.js' %}"></script>
+{% load render_bundle from webpack_loader %}
+
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>OSF Admin | Dashboard</title>
+    <!-- Tell the browser to be responsive to screen width -->
+    <meta content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no' name='viewport'>
+    <!-- Bootstrap 3.3.4 -->
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet" type="text/css" />
+    <!-- Font Awesome Icons -->
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
```

Comment:
You're going to be better off using protocol independent urls, eg: 
```html
<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
```

Right now, on the dev server which isn't https, you get warnings from some browsers about requesting http assets. Some browsers won't include them at all which isn't fun to debug. The protocol independent urls are like relative urls, they'll just use whatever protocol the initial request was made with.